File: pdlearning/src/mobile-apps/LearnerApp/Models/Course/Comment.cs
using System;
using Newtonsoft.Json;

namespace LearnerApp.Models.Course
{
    public class Comment
    {
        public string Id { get; set; }

        public string ObjectId { get; set; }

        public string UserId { get; set; }

        public string Content { get; set; }

        public DateTime CreatedDate { get; set; }

        public DateTime? ChangedDate { get; set; }

        public string Action { get; set; }

        [JsonIgnore]
        public string AvatarUrl { get; set; }

        [JsonIgnore]
        public string FullName { get; set; }

        [JsonIgnore]
        public string DepartmentName { get; set; }
    }
}

File: pdlearning/src/web-apps/src/libraries/domain-api/src/comment/models/comment.ts
export interface IComment {
  id: string;
  content: string;
  userId: string;
  objectId: string;
  createdDate: string;
  changedDate: string;
  action?: string;
}

export class Comment implements IComment {
  public id: string;
  public content: string;
  public userId: string;
  public objectId: string;
  public createdDate: string;
  public changedDate: string;
  public action?: string;
  constructor(data: IComment) {
    this.id = data.id;
    this.id = data.id;
    this.content = data.content;
    this.userId = data.userId;
    this.objectId = data.objectId;
    this.createdDate = data.createdDate;
    this.changedDate = data.changedDate;
    this.action = data.action;
  }
}
